examples/xgboost/xgboost_sklearn/train_sklearn.py

```diff
@@ -0,0 +1,44 @@
+from pprint import pprint
+
+import pandas as pd
+import xgboost as xgb
+from sklearn.datasets import load_diabetes
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import mean_squared_error
+
+import numpy as np
+import mlflow
+import mlflow.xgboost
+
+from utils import fetch_logged_data
+
+
+def main():
+    # prepare example dataset
+    diabetes = load_diabetes()
+    X = pd.DataFrame(diabetes.data, columns=diabetes.feature_names)
+    y = pd.Series(diabetes.target)
```

Comment:
```suggestion
    X, y = load_diabetes(return_X_y=True, as_frame=True)
```

Let's use `return_X_y` and `as_frame` to simplify the code.